torch/csrc/api/include/torch/special.h

```diff
@@ -69,6 +69,22 @@ inline Tensor& erfc_out(Tensor& result, const Tensor& self) {
   return torch::special_erfc_out(result, self);
 }
 
+/// Computes the scaled complementary error function
+/// See https://pytorch.org/docs/master/special.html#torch.special.erfcx.
+///
+/// Example:
+/// ```
+/// auto t = torch::randn(128, dtype=kDouble);
+/// torch::special::erfcx(t);
+/// ```
+inline Tensor erfcx(const Tensor& self) {
+  return torch::special_erfcx(self);
+}
+
+inline Tensor erfcx_out(const Tensor& self) {
```

Comment:
`inline Tensor& erfcx_out(Tensor& result, const Tensor& self)` 

I mess this up sometimes :sweat_smile: